src/app/lib/config/services/afaanoromoo.js

```diff
@@ -129,39 +129,31 @@ const service = {
   footer: {
     externalLink: {
       href: 'https://www.bbc.co.uk/help/web/links/',
-      text: 'Read about our approach to external linking.',
+      text: 'Itti dhiyaachuu keenya gara geessituu.',
     },
     links: [
       {
-        href: 'https://www.bbc.com/news/help-41670342',
-        text: 'Why you can trust the BBC',
+        href: 'https://www.bbc.com/afaanoromoo/institutional-49281861',
+        text: 'BBC News maaliif amanuu dandeessa',
       },
       {
-        href: 'https://www.bbc.com/terms',
-        text: 'Terms of Use',
+        href: 'https://www.bbc.co.uk/usingthebbc/terms/',
+        text: 'Haala itti fayyadamaa',
       },
       {
-        href: 'https://www.bbc.co.uk/aboutthebbc/',
-        text: 'About the BBC',
-      },
-      {
-        href: 'https://www.bbc.com/privacy/',
-        text: 'Privacy Policy',
+        href: 'https://www.bbc.co.uk/usingthebbc/privacy/',
+        text: 'Imaammata mateenyaa',
       },
       {
         href: 'https://www.bbc.com/usingthebbc/cookies/',
-        text: 'Cookies',
-      },
-      {
-        href: 'https://www.bbc.com/accessibility/',
```

Comment:
Similarly to `aboutthebbc` is the `accessibility` link being dropped?

Reply:
Can you confirm if these links are being removed on purpose? If so I'll approve

Reply:
Seems to be intentional - @aeroplanejane confirmed at standup.  See notes / Jane's comment here: https://github.com/bbc/simorgh/issues/3034